frontend/style.css

```diff
@@ -13,9 +13,9 @@ body {
 
 #divForm {
     display: grid;
-    grid-template-columns: 180px 200px;
+    grid-template-columns: 130px 250px;
     grid-template-rows: 42px 42px 42px 42px 42px 42px 42px 42px 42px;
-    justify-items: start;
+    /* justify-items: start; */
```

Comment:
Remove the comments. If you want to keep it as reference, then either save it somewhere else or using stage keep it locally.